quickstarts/Atlassian JIRA

```diff
@@ -0,0 +1,46 @@
+id: c5c5dd30-dcdf-46b6-9412-f9a1bba5a600
+# Name of the quickstart (required)
+name: atlassian-jira
+
+# Displayed in the UI (required)
+title: Atlassian JIRA
+
+# Long-form description of the quickstart (required)
+description: |
+  Jira is a proprietary issue tracking product developed by Atlassian that allows bug tracking and agile project management. You can Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
+  
+  For more information or support, please go to https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#jira
+
+
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+summary: |
+  Integrate New Relic with Atlassian Jira(Cloud) and automatically create and update Jira issues.
+
+# Support level: New Relic | Verified | Community (required)
+level: Verified
+
+# Authors of the quickstart (required)
+authors:
+  - Matan Moser (New Relic)
+  - Ismail Azam (New Relic)
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - JIRA
+  - Atlassian
+  - Notifications
+  - AIOps
+  - Incident Intelligence
+  - Destinations
+
+installPlans:
+  - third-party-JIRA
```

Comment:
```suggestion
  - third-party-jira
```

We will also need to create an install plan in the `installs` folder which I can show you how to do.